.circleci/config.yml

```diff
@@ -3,12 +3,33 @@ version: 2.1
 orbs:
   hmpps: ministryofjustice/hmpps@3.1
 
+executors:
+  validator:
```

Comment:
A shame we had to go back to defining the exectuor but the orb doesn't support multiple supporting images.